HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h

```diff
@@ -86,15 +82,20 @@ namespace cms {
       uint32_t *off = (uint32_t *)((char *)(h) + offsetof(Histo, off));
       size_t wss = Histo::wsSize();
       assert(wss > 0);
-      CubDebugExit(cub::DeviceScan::InclusiveSum(ws, wss, off, off, Histo::totbins(), stream));
+      auto nthreads = 512+256;
```

Comment:
is this still valid, considering the comment above on relaxing this limit (after removing the `-G` flag) ?

Reply:
most probably not. can be 1024 (not sure that is more efficent)

Reply:
Something similar is present also in the VertexFinder.
In general I think we agreed that at some point we should find a "clever" way to optmize all these block sizes

Reply:
Right, that's true.
OK, if this doesn't impact performance we can keep it as is until we get there :-)